src/ducks/registry/index.js

```diff
@@ -25,4 +27,24 @@ export const getRegistryKonnector = (state, slug) => state.registry.konnectors.f
 
 // action creators sync
 
-export const initializeRegistry = konnectors => ({type: INITIALIZE_REGISTRY_KONNECTORS, konnectors: konnectors})
+let contextCache
```

Comment:
Do we need to cache the context here and now ? Isn't a CozyClient's concern instead ?

Reply:
We don't really need since this parti is not used yet. I can remove it for now so.

Reply:
Feel free, it was just a question, it does not bother me. Maybe if not useful we may remove it to keep the code as "just what we need"